frontend/src/components/CreateCollectionModal/components/Content/index.tsx

```diff
@@ -82,11 +83,13 @@ const Content: FC<Props> = (props) => {
 
   const formEl = useRef<HTMLFormElement>(null);
 
-  const { data } = useCollection({
+  let { data } = useCollection({
     id: props.id,
     visibility: VISIBILITY_TYPE.PRIVATE,
   });
 
+  if (isTombstonedCollection(data)) data = {} as Collection;
```

Comment:
instead of assigning a new POJO every time, maybe `null` would work here? Otherwise `{} !== {}` would cause unnecessary re-rendering

Also reassigning variable to a different value impacts readability and is a slippery slope if future authors continue to follow the pattern to reassign the value through out the file

Maybe we can change it to:

```ts
const { data } = useCollection({
    id: props.id,
    visibility: VISIBILITY_TYPE.PRIVATE,
  });

const collection = isTombstonedCollection(data) ? null : data;

// and then use `collection` for the rest of the file?
```

Reply:
Sounds good, should I change line 96 too?

https://github.com/chanzuckerberg/single-cell-data-portal/blob/723b97039f34c503f97e0f7bacb08781d1f4968b/frontend/src/components/CreateCollectionModal/components/Content/index.tsx#L96

Reply:
Great catch! Yes please!

Reply:
I'm actually going to stick with the original here.
https://github.com/chanzuckerberg/single-cell-data-portal/blob/723b97039f34c503f97e0f7bacb08781d1f4968b/frontend/src/components/CreateCollectionModal/components/Content/index.tsx#L96

We shouldn't really be getting to the case that the create collection modal is open on a tombstoned collection.  Too much of an edge case to justify a tsignore/ the refactor necessary.

![image](https://user-images.githubusercontent.com/8716829/141836957-ba1b83d0-47ec-4daf-a86a-06e21fd1d5c6.png)

Reply:
Hmm yeah that sounds great 👍 Thanks for investigating this approach!